cmd/offboard/offboard.go

```diff
@@ -0,0 +1,101 @@
+package offboard
+
+import (
+	"errors"
+	"fmt"
+	"slices"
+
+	"github.com/spf13/cobra"
+
+	"github.com/open-sauced/pizza-cli/v2/pkg/config"
+)
+
+type Options struct {
+	offboardingUsers []string
+
+	// config file path
+	configPath string
+
+	// repository path
+	path string
+
+	// from global config
+	ttyDisabled bool
+}
+
+const offboardLongDesc string = `[WIP] Removes a user from the \".sauced.yaml\" config and \"CODEOWNERS\" files.
+Requires the user's name OR email.`
+
+func NewConfigCommand() *cobra.Command {
+	opts := &Options{}
+	cmd := &cobra.Command{
+		Use:   "offboard <username/email> [flags]",
+		Short: "[WIP] Removes a user from the \".sauced.yaml\" config and \"CODEOWNERS\" files.",
+		Long:  offboardLongDesc,
+		Args: func(_ *cobra.Command, args []string) error {
+			if !(len(args) > 0) {
+				return errors.New("you must provide at least one argument: the offboarding user's email/username")
+			}
+
+			opts.offboardingUsers = args
+
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			opts.ttyDisabled, _ = cmd.Flags().GetBool("tty-disable")
+			opts.configPath, _ = cmd.Flags().GetString("config")
+
+			opts.path, _ = cmd.Flags().GetString("path")
```

Comment:
This flag likely needs to get marked as required. Otherwise, it attempts to use the default value:

```
❯ pizza offboard jpmcb
Error: error generating config file: error creating  file: open : no such file or directory
```